web/docusaurus/src/sidebars.js

```diff
@@ -5,39 +5,6 @@ const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
   welcomeSidebar: [
     {type: 'autogenerated', dirName: '.'},
-    {
-      type: 'category',
-      label: 'Manuals',
-      collapsible: true,
-      collapsed: true,
-      items: [
-        {
-          type: 'link',
-          label: 'nodeos',
-          href: '/leap/latest/nodeos/'
-        },
-        {
-          type: 'link',
-          label: 'cleos',
-          href: '/leap/latest/cleos/'
-        },
-        {
-          type: 'link',
-          label: 'keosd',
-          href: '/leap/latest/keosd/'
-        },
-        {
-          type: 'link',
-          label: 'CDT',
-          href: '/cdt/latest/'
-        },
-        {
-          type: 'link',
-          label: 'System Contracts',
-          href: '/system-contracts/latest/'
-        },
-      ],
-    },
     {type: 'link', label: 'DUNE', href: '/DUNE/latest/', },
```

Comment:
Should delete DUNE, a manual add to the sidebar

Reply:
100% thanks for pointing this out, I will move it from the sidebar to the top navigation menu